huaweicloud/services/cce/resource_huaweicloud_cce_node.go

```diff
@@ -1269,32 +1305,18 @@ func getResourceIDFromJob(ctx context.Context, client *golangsdk.ServiceClient,
 	return nodeid, nil
```

Comment:
for _, s := range job.Spec.SubJobs {
		if s.Spec.Type == subJobType {
			return s.Spec.ResourceID, nil
		}
	}
	return "", fmt.Errorf("error fetching %s Job resource id", subJobType)

Reply:
sure